the-collab-lab/tcl-71-smart-shopping-list
# Style the home page 'All my lists'

## Description

This PR implements styling, as defined by the [Figma](https://www.figma.com/file/zPeqMaa7CZs1lewYuNWat6/tcl-71-smart-shopping-list?type=design&node-id=0-1&mode=design&t=ropZExMhIYBX4Y0c-0) document, to the Home view, which includes components SingleList and ListForm.

There are two things for consideration about this PR:

1- Font sizing, font family, and outer spacing have been implemented at the component level, but that could change if these are standarized in an upper component.
2- List name overflow has been handled by hiding excesively long names to not affect the layout, but this probably should be handled by setting a character limit on creation.

## Related Issue

closes #46 

## Acceptance Criteria

- [x] Fonts and colors are respected.
- [x] The icons have the correct color.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|  ✓    | :art:  Enhancement |

## Updates

### Before

![image](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/86715948/20c4186b-f28f-4c05-945c-50047fbb1ade)

### After

![image](https://github.com/the-collab-lab/tcl-71-smart-shopping-list/assets/86715948/b46fabd1-17a3-4f51-9136-cec053b93473)

## Testing Steps / QA Criteria

- `git pull`
- `git checkout bm-style-home`
- Switching between lists should show the new Loading spinner instead of the old list items while the list loads.
